messaging-experiments.json

```diff
@@ -348,5 +348,156 @@
         }
       ]
     }
+  },
+  {
+    "id": "MULTI-STAGE-ABOUTWELCOME-V1-78-BETA-V2",
+    "enabled": true,
+    "filter_expression": "env.version >= '78.' && env.version < '79.' && env.channel == 'beta' && env.appinfo.appBuildID >= '20200610093716'",
+    "targeting": "localeLanguageCode == \"en\" && [userId, \"bug-1644521-message-multi-stage-aboutwelcome-v1-78-beta-v2-beta-78-79\"]|bucketSample(0, 5000, 10000) && (isFirstStartup || 'bug-1644521-message-multi-stage-aboutwelcome-v1-78-beta-v2-beta-78-79' in activeExperiments)",
+    "arguments": {
+      "slug": "bug-1644521-message-multi-stage-aboutwelcome-v1-78-beta-v2-beta-78-79",
+      "userFacingName": "About:Welcome MultiStage Welcome Beta",
+      "experimentDocumentUrl": "https://experimenter.services.mozilla.com/experiments/multi-stage-aboutwelcome-v1-78-beta-v2/",
+      "userFacingDescription": "Study to validate launching multistage welcome flow via experiment framework.",
+      "isEnrollmentPaused": false,
+      "branches": [
+        {
+          "slug": "control",
+          "ratio": 1,
+          "groups": [
+            "aboutwelcome"
+          ],
+          "value": {}
+        },
+        {
+          "slug": "treatment-static",
+          "ratio": 1,
+          "groups": [
+            "aboutwelcome"
+          ],
+          "value": {
+            "id": "multi-stage-welcome",
+            "template": "multistage",
+            "screens": [
+              {
+                "id": "AW_GET_STARTED",
+                "order": 0,
+                "content": {
+                  "title": {
+                    "string_id": "onboarding-welcome-header"
+                  },
+                  "subtitle": "The fast, safe, and private browser that's backed by a non-profit.",
+                  "primary_button": {
+                    "label": "Start Setup",
+                    "action": {
+                      "navigate": true
+                    }
+                  },
+                  "secondary_button": {
+                    "text": "Have an account?",
+                    "label": "Sign in",
+                    "position": "top",
+                    "action": {
+                      "type": "OPEN_URL",
+                      "addFlowParams": true,
+                      "data": {
+                        "args": "https://accounts.firefox.com/?service=sync&action=email&context=fx_desktop_v3&entrypoint=activity-stream-firstrun&style=trailhead",
+                        "where": "current"
+                      }
+                    }
+                  }
+                }
+              },
+              {
+                "id": "AW_IMPORT_SETTINGS",
+                "order": 1,
+                "content": {
+                  "title": "Import your passwords, bookmarks, and more",
```

Comment:
NI @Mardak to help update content here with non breaking spaces that display bookmarks, and more in its own line

Reply:
Type in a nonbreaking space after the last comma and before more. You can press <alt>+<space> to make one on macOS or you can put in `\xa0` for `… bookmarks,\xa0and\xa0more"`